jboss-image-streams.json

```diff
@@ -554,35 +564,35 @@
                         "annotations": {
                             "description": "JBoss EAP 7.1 Tech Preview.",
                             "iconClass": "icon-jboss",
-                            "tags": "builder,eap,javaee,java,jboss,xpaas",
-                            "supports":"eap:7.1,javaee:7,java:8,xpaas:1.0",
+                            "tags": "builder,eap,javaee,java,jboss,hidden",
+                            "supports": "eap:7.1,javaee:7,java:8:1.0",
                             "sampleRepo": "https://github.com/jboss-developer/jboss-eap-quickstarts.git",
                             "sampleContextDir": "kitchensink",
                             "sampleRef": "7.0.0.GA",
                             "version": "TP",
                             "openshift.io/display-name": "Red Hat JBoss EAP 7.1 (Tech Preview)"
                         },
                         "from": {
-                          "kind": "ImageStreamTag",
-                          "name": "1.1-TP"
+                            "kind": "ImageStreamTag",
+                            "name": "1.0-TP"
```

Comment:
This change seems unrelated to the work being done; possibly a merge error?

Reply:
I fixed this, it was wrongly pointing to 1.1-TP, which doesn't exist. I could make the change elsewhere but I noticed this error. See line 571 from which defines 1.0-TP not 1.1.